component-samples/demo/manufacturer/manufacturer.env

```diff
@@ -6,6 +6,15 @@ manufacturer_database_port=8049
 catalina_home=./target/tomcat
 manufacturer_api_user=apiUser
 
+## PRI Manufacturer onDie Congfigurations
+## Uncomment the following lines to enable onDie support
+
+#ondie_cache=file:///home/manufacturer/ondie_cache/
```

Comment:
The user name is 'fdo' for all components. 
ondie_cache=file:///home/fdo/ondie_cache/

Reply:
Done.